include/message_filters/subscriber.h

```diff
@@ -152,8 +146,8 @@ struct message_type <M, true>
   using type = typename M::custom_type;
 };
 
-template <typename M> 
-struct message_type <M, false> 
+template <typename M>
+struct message_type <M, false>
```

Comment:
I'm going to suggest we not make this change here.  In other words, I think this PR should strictly be about copyright; leave these other fixes to other PRs when we introduce the code linters.